binr/rafind2/rafind2.c

```diff
@@ -81,14 +82,15 @@ static int hit(RSearchKeyword *kw, void *user, ut64 addr) {
 }
 
 static int show_help(char *argv0, int line) {
-	printf ("Usage: %s [-mXnzZhv] [-a align] [-b sz] [-f/t from/to] [-[e|s|S] str] [-x hex] file ..\n", argv0);
+	printf ("Usage: %s [-mXnzZhHv] [-a align] [-b sz] [-f/t from/to] [-[e|s|S] str] [-x hex] file|dir ..\n", argv0);
 	if (line) return 0;
 	printf (
 	" -a [align] only accept aligned hits\n"
 	" -b [size]  set block size\n"
 	" -e [regex] search for regex matches (can be used multiple times)\n"
 	" -f [from]  start searching from address 'from'\n"
 	" -h         show this help\n"
+	" -H         do not show headings (filenames) above matching contents (default for searching a single file)\n"
```

Comment:
i still prefer the -q, as quiet, because it's a flag used in all the other r2 commands

Reply:
Ok, sure! Should it suppress any other messages too then? I suppressing the headings by default if only one file is used ok too?